### PR TITLE
docs(arcor2_arserver): update lock readme

### PR DESCRIPTION
**Pull request by denny101603**
info about RPCs with automatical unlocking of objects

### CODE REVIEW

File: src/python/arcor2_arserver/lock/README.md [hunk 1]
```diff
@@ -30,6 +30,15 @@
 
 ### User lock maintanance
 
-- Use can generally edit projects and use locks after login.
+- User can generally edit projects and use locks after login.
```
Comment (ZdenekM) [inline src/python/arcor2_arserver/lock/README.md:33]: A user

File: src/python/arcor2_arserver/lock/README.md [hunk 2]
```diff
@@ -30,6 +30,15 @@
...
 - When the user logs out with some acquired locks, timeout is executed. After the timeout period all affected locks are released.
   - When user activity is restored, the cleaning task is canceled.
+
+
+### RPCs with non-standard behavior
+- Rename: After renaming an action object, the object is automatically unlocked by server.
```
Comment (ZdenekM) [inline src/python/arcor2_arserver/lock/README.md:39]: The RPC is `RenameObject`.

File: src/python/arcor2_arserver/lock/README.md [hunk 3]
```diff
@@ -30,6 +30,15 @@
...
+- UpdateActionPointParent: When updating parent of action point, only the child has to be locked manually. The parent is locked automatically by server and then both child and parent are unlocked automatically by server.
```
Comment (ZdenekM) [inline src/python/arcor2_arserver/lock/README.md:40]: Please format RPC names as code (with "`").

**Review by ZdenekM — CHANGES_REQUESTED** [review]

**Comment by denny101603** [comment]
going to open new PR with squashed commits

**Comment by ZdenekM** [comment]
You could let it openned, squash commits, and then force push. The existing PR will be updated.

**Pull request closed without merging**